set.mm

```diff
@@ -534224,22 +534252,14 @@ Identity relation (complements)
   bj-elid4 $p |- ( <. x , A >. e. _I <-> x = A ) $=
     ( cv cop cid wcel cvv wceq vex bj-elid3 mpbiran ) ACZBDEFLGFLBHAILBJK $.
 
-$(
-  ${
-    @( The intersection of the identity relation with a cartesian product is a
-       restricted identity relation.  (Contributed by BJ, 12-Sep-2022.) @)
-    idinxprid $p |- ( _I i^i ( A X. B ) ) = ( _I |` ( A i^i B ) ) $=
-      ? $.
-  $}
-$)
-
 
 $(
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
-  Diagonal in a Cartesian square
+  Functionalized identity relation (diagonal in a Cartesian square)
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 
-  Definition of the diagonal in the Cartesian square of a set.
+  This subsection defines a functionalized  version of the identity relation,
```

Comment:
```suggestion
  This subsection defines a functionalized version of the identity relation,
```

[mathbox]

Reply:
I cannot apply the suggestion on this webpage. If anyone can, please do. I've already done it on my local branch but there are several more changes and I'd like to do that in an upcoming PR.